docs/content/concepts/entity-path.md

```diff
@@ -4,26 +4,28 @@ order: 1
 ---
 
 ## Entity Paths
-As mentioned in the [Entity Component](entity-component.md) overview, all Entities within Rerun have a unique _Entity Path_.
+As mentioned in the [Entity Component](entity-component.md) overview, all entities within Rerun have a unique _entity path_.
 
-The first argument to each log function is this path. Each time you log to a specific entity path you will update the entity, i.e. log a new instance of it along the timeline. Each logging to a path must be of the same type (you cannot log an image to the same path as a point cloud).
+The first argument to the [`rr.log()`](https://ref.rerun.io/docs/python/HEAD/common/logging/#rerun.log) function is this path. Each time you log to a specific entity path you will update the entity, i.e. log a new instance of it along the timeline.
 
-Rerun treats these paths as being arranged in a hierarchy with the "/" character acting as a separator between path
-elements. The conventional path semantics including concepts of "root" and "parent" / "child" generally apply.
+Note that although it is technically possible to log multiple types of archetypes on the same entity path, it's almost certainly not what you want. For example, logging a [`Points3D`](../reference/data_types/archetypes/points3d.md) point cloud on an entity path where a [`Mesh3D`](../reference/data_types/archetypes/mesh3d.md) was previously logged would overwrite the mesh's [`Position3D`](../reference/data_types/components/position3d.md) component with the point cloud's, but would leave the [`MeshProperties`](../reference/data_types/components/mesh_properties.md) component untouched. The Rerun viewer would likely be unable to display the result. See the [Entity Component](entity-component.md) section for more information.
```

Comment:
`it's in many cases not what you want`

The problem here is that we force you to do that for Transforms3D and a few others

Reply:
Good call. I've clarified that.